### PR TITLE
Add feat config and debug (first part)

### PR DESCRIPTION
**Pull request by rloredo**
Add command config to create the yml file with Jira config.
Add command debug  to test if the file exists.
The function to load the config as a dictionary was also implemented.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 env
 __pycache__
 lib
+*.egg-info
```
Comment (muripic) [inline .gitignore]: Add newline at end of file

File: README.md [hunk 2]
```diff
@@ -2,10 +2,20 @@
...
 
-## Development
+The easiest (and best) way to install git-jira is through pip:
 
+```bash
+pip install git-jira
 ```
+
+To make it work, you need a `config.yml` with your authentication credentials and default project.  
+For that, run `git jira config`.
```
Comment (muripic) [inline README.md:14]:
```suggestion
Run `git jira config` to generate this file.
```

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,6 @@
...
+build
+dist
```
Comment (muripic) [inline .gitignore:6]: new line at end of file missing

File: git_jira/cli.py [hunk 4]
```diff
@@ -1,28 +1,41 @@
...
     pass
 
+@click.command()
+def config():
+    click.echo("Welcome to git-jira. Let's generate your config file...")
+    jira_server_url = click.prompt("Insert your Jira server url", type=str)
+    jira_username = click.prompt("Insert your Jira username", type=str)
+    jira_api_token = click.prompt("Create an API token here https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/ \nand insert your Jira API token", type=str)
+    jira_default_project_key = click.prompt("Select default project to use. \nLeave empty if you want to specify it as flags. Insert the short code (like TP, PJ, etc)", type=str) 
+    click.echo(f"A config.yaml file with the following params will be created at {CONFIG_FILE_PATH}\n\n{config_params_to_str(jira_server_url, jira_username, jira_api_token, jira_default_project_key)}\n")
```
Comment (muripic) [inline git_jira/cli.py:15]: If you create a dictionary with these keys, you can then use `yaml.dump(configs)` and get the yaml output.

**Comment by rloredo** [comment]
Add command config to create the yml file with Jira config.
Add command debug to test if the file exists.
The function to load the config as a dictionary was also implemented.

**Pull request merged**